src/Atoms/ButtonsExercises/ButtonsExercises.js

```diff
@@ -0,0 +1,38 @@
+import ButtonExercisesStyled from './ButtonExercisesStyled';
+import { FaChevronLeft } from 'react-icons/fa';
+import propTypes from 'prop-types';
+import { useNavigate } from 'react-router-dom';
+
+const ButtonExercises = (props) => {
+  const { endingButton, goBack, text } = props;
+  const navigate = useNavigate();
+
+  const handleClick = () => {
+    if (goBack) {
+      navigate(-1);
+    } else {
+      navigate('./dogSummary', {
+        state: { text: 'Tabela Wyników', label: 'Ocena Zawodnika' },
+      });
+    }
+  };
+  return (
+    <ButtonExercisesStyled
+      onClick={handleClick}
+      endingButton={endingButton}
+      goBack={goBack}
```

Comment:
Related to https://github.com/CC2021-WBL/GOOD-BOI-Application/pull/202/files#r798045688, this shuold receive only information about what theme color you want to apply. color={endingButton? "primary":"secondary"}

Reply:
- names fixed